nextjs-13/src/components/NewsletterItem/index.tsx

```diff
@@ -0,0 +1,61 @@
+import Image from 'next/image';
+import Swal from 'sweetalert2';
```

Comment:
As done for Remix (see https://github.com/nimblehq/js-framework-benchmark/blob/develop/remix/app/lib/notification/showNotification.ts), it would be best to abstract the notification implementation (i..e, not having to import `sweetalert2` in more than one place). But if it is too much work we can level it as-is for this project.

Reply:
Resolved [here](https://github.com/nimblehq/js-framework-benchmark/pull/116/commits/75f2f57cc9ffc0d5e0b786fbb3c3f9c1c1db3586)

Reply:
It is better but as recommended to Beckham, it would be better to use a more generic name e.g., `alert`, `notification`, etc. We don't need to stick to the library name 😅 What do you think?

Reply:
Ah sure! It can be abstracted more 👀 

Improved [here](https://github.com/nimblehq/js-framework-benchmark/pull/116/commits/eb5b553f9b6d154c8e85ca86a43ad30860b4972c)